source/main/windows/main.ts

```diff
@@ -38,12 +38,15 @@ type WindowOptionsType = {
   };
   icon?: string;
 };
-export const createMainWindow = (locale: string, windowBounds?: Rectangle) => {
+export const createMainWindow = (
```

Comment:
Why isn’t this instance changed to the new (old?) `function createMainWindow(locale…` syntax?

Reply:
Good question, need to check if that is somehow excluded from the checks

Reply:
@michalrus i think that's because they other changes are required by some react-specific linting rule that suggests using named functions because it is then easier to debug react components in the browser (because they have names that show up in the React developer tools etc.)